apps/fm/js/fm.js

```diff
@@ -158,6 +158,9 @@ var mozFMRadio = navigator.mozFM || navigator.mozFMRadio || {
   aGlobal.SpeakerManager = SpeakerManager;
 })(window);
 
+// the mhzString that will be localized.
+var mhzString = 'MHz-NOTLOCALIZED';
```

Comment:
Why do you at all initialize this string here? It seems that it should not be used outside of the `formatValue` call in the `DOMLocalized` handler?